lms/djangoapps/teams/static/teams/js/views/team_utils.js

```diff
@@ -0,0 +1,39 @@
+/*  Team utility methods*/
+;(function (define) {
+    'use strict';
+    define([
+        'backbone'
+    ], function (Backbone) {
+        return _.extend(Backbone.View.prototype, {
```

Comment:
Why do we need to extend `Backbone.View` here? Shouldn't returning a clean dict be sufficient?